.github/workflows/ci.yaml

```diff
@@ -19,15 +19,54 @@ jobs:
       matrix:
         folder:
           [
-            "packages/rx_bloc_list",
-            "packages/flutter_rx_bloc",
             "examples/booking_app",
             "examples/counter",
             "examples/division",
             "examples/favorites_advanced/rx_bloc_favorites_advanced",
           ]
       fail-fast: false
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: subosito/flutter-action@v1.5.3
+        with:
+          channel: "stable"
+      - run: dart --version
+      - run: flutter --version 
+      - name: Install Dependencies
+        working-directory: ${{ matrix.folder }}
+        run: flutter pub get
+      - name: Analyze
+        working-directory: ${{ matrix.folder }}
+        run: flutter analyze lib
+      - name: Run tests
+        working-directory: ${{ matrix.folder }}
+        run: |
+          if [ -d "test" ]; then
+            flutter test --coverage --exclude-tags=not-tests
+          fi
+      - name: Remove unnecessary files
+        working-directory: ${{ matrix.folder }}
+        run: flutter pub run clean_coverage clean --exclusions '**/.g.dart','**repository.dart','**rxb.g.dart' coverage/lcov.info
```

Comment:
Just FYI, I have a python script that can do the same thing much faster. We can use it if the pipeline runtime is an issue and needs to be reduced.

Reply:
Good to know. As of now we have no issues with the pipeline duration, but if we face such an issue we can look in this option.